backend/src/main/java/org/cryptomator/hub/api/VaultResource.java

```diff
@@ -60,6 +64,16 @@ public List<VaultDto> getSharedOrOwned() {
 		return resultStream.map(VaultDto::fromEntity).toList();
 	}
 
+	@GET
+	@Path("/all")
```

Comment:
Strictly speaking this is not a different resource, just a different filter. Is it possible to rather use `/api/vaults?all` without much hassle? If this requires adding a `@QueryParam` to the existing `getSharedOrOwned()` and requiring complicated branching depending on roles, I would prefer the current `/api/vaults/all` approach, though.

Reply:
Did some research and found no way to provide the correct response without branching or a request filter. At least the role must be checked manually because all vaults should only be called from the admin role.